.TH "jpl::_exception::FileNotFoundException" 3Version 1.0.0" "Exception" \" -*- nroff -*-
.ad l
.nh
.SH NAME
jpl::_exception::FileNotFoundException
.SH SYNOPSIS
.br
.PP
.PP
\fC#include <FileNotFoundException\&.hpp>\fP
.PP
Inherits \fBjpl::_exception::RuntimeException\fP\&.
.SS "Public Member Functions"

.in +1c
.ti -1c
.RI "\fBFileNotFoundException\fP (std::string \fBmsg\fP)"
.br
.ti -1c
.RI "\fBFileNotFoundException\fP ()"
.br
.in -1c

Public Member Functions inherited from \fBjpl::_exception::RuntimeException\fP
.in +1c
.ti -1c
.RI "\fBRuntimeException\fP (std::string \fBtype_ex\fP, std::string \fBmsg\fP)"
.br
.ti -1c
.RI "\fBRuntimeException\fP (std::string \fBmsg\fP)"
.br
.ti -1c
.RI "\fBRuntimeException\fP ()"
.br
.in -1c

Public Member Functions inherited from \fBjpl::_exception::AbstractException\fP
.in +1c
.ti -1c
.RI "virtual const char * \fBwhat\fP () const _GLIBCXX_TXN_SAFE_DYN _GLIBCXX_NOTHROW override"
.br
.RI "Print the type of the exception which has been thrown with the message that has been passed, too\&. "
.in -1c
.SS "Additional Inherited Members"


Protected Member Functions inherited from \fBjpl::_exception::AbstractException\fP
.in +1c
.ti -1c
.RI "\fBAbstractException\fP (std::string \fBtype_ex\fP)"
.br
.ti -1c
.RI "\fBAbstractException\fP (std::string \fBtype_ex\fP, std::string \fBmsg\fP)"
.br
.in -1c

Protected Attributes inherited from \fBjpl::_exception::AbstractException\fP
.in +1c
.ti -1c
.RI "std::string \fBtype_ex\fP"
.br
.ti -1c
.RI "std::string \fBmsg\fP"
.br
.in -1c
.SH "Detailed Description"
.PP 
Definition at line \fB20\fP of file \fBFileNotFoundException\&.hpp\fP\&.
.SH "Constructor & Destructor Documentation"
.PP 
.SS "jpl::_exception::FileNotFoundException::FileNotFoundException (std::string msg)\fC [inline]\fP"

.PP
Definition at line \fB25\fP of file \fBFileNotFoundException\&.hpp\fP\&.
.SS "jpl::_exception::FileNotFoundException::FileNotFoundException ()\fC [inline]\fP"

.PP
Definition at line \fB26\fP of file \fBFileNotFoundException\&.hpp\fP\&.

.SH "Author"
.PP 
Generated automatically by Doxygen for Exception from the source code\&.
